projects/demo/src/modules/markup/icons/icons.style.less

```diff
@@ -2,5 +2,32 @@
 
 .icon {
     margin: @space * 3;
-    border-radius: 0;
+    transition: all 0.5s ease-in;
+    background-image: linear-gradient(45deg, var(--tui-base-02) 25%, transparent 25%),
```

Comment:
I have doubts about this CSS-property.
Do we actually need this "pseudo-trasparent"-effect ?

<img width="269" alt="Screenshot 2023-04-21 at 16 43 16" src="https://user-images.githubusercontent.com/35179038/233651403-d85be2b5-b9f0-41b8-9bbc-effc04ca74de.png">

It is difficult to "read" icons. Maybe let's keeps everything except this `background-image` ?
<img width="269" alt="Screenshot 2023-04-21 at 16 43 03" src="https://user-images.githubusercontent.com/35179038/233651441-69831402-ef5d-4823-b87c-b6b3cbb785f5.png">

Reply:
see my comments in time chat